Tools/GraphGeneration/README.md

```diff
@@ -0,0 +1,67 @@
+
+## Purpose
+The main objective is to create synthetic star graphs representing emergency response systems with:
+- PSAPs at the center.
+- Responders and callers distributed within specified regions.
+- Special handling for prime numbers of PSAPs and other specific cases.
+- The `visualize_generation.py` script specifically focuses on visualizing the generation of a synthetic network graph from parameterized GraphML data, highlighting different node types and their positions within defined regions.
+
+## graph_generation folder 
+
+### `graph_generation.py`
+Generates synthetic star graphs based on input parameters. The script handles various scenarios, including special cases such as prime numbers of PSAPs, to ensure appropriate distribution of graph elements. It includes logic for:
+ - Distributing PSAPs, responders, and callers within defined regions.
+ - Handling prime numbers of PSAPs by creating an extra region and combining it appropriately.
+ - Ensuring each region contains a star-shaped graph centered on a PSAP.
+
+### `visualize_generation.py`
+Visualizes the graphs generated by `graph_generation.py` from the GraphML file. It displays the graph with color-coded nodes for PSAPs, callers, and responders. It outlines the rectangular boundary, PSAP regions, and caller regions, allowing for easy analysis and verification of the graph structure.
+
+### `format_graphml.py`
+Converts the original synthetic GraphML file to a format that can be used within the cluster-point call generation and Graphitti simulations. This script ensures the GraphML file is compatible with the tools and configurations used in subsequent simulation steps.
+
+## call_generation folder 
+Contains scripts for generating call data from the GraphML files created in the `graphml_generation` folder. These scripts use the structured graphs to simulate call events, providing data for emergency response simulations.
+
+## config_files folder 
+Holds configuration files for GraphGeneration Graphitti simulations. These files use the generated graphs and call data to configure and run various simulation scenarios, testing different aspects of emergency response systems.
+
+## graphml_generation folder 
+Stores the generated graph models from `graph_generation.py`, formatted through `format_graphml.py`. These GraphML files serve as the basis for further simulations and call generation, ensuring consistency and compatibility across different simulation tools.
+
+## output_files folder 
+Contains the output files from the simulations run using the configuration files in the `config_files` folder. These outputs include results and logs from the simulation runs, which can be analyzed to evaluate the performance and effectiveness of the generated graphs and configurations.
```

Comment:
Don't include data in this repository. If you want to include examples because you think it's important for their explanatory value, then make an `examples` directory. File names should be more explanatory; `synth` in particular isn't.